apps/studio/src/pages/sign-in/singpass/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { useRouter } from "next/router"
+import { Flex, Grid, GridItem, Text, useDisclosure } from "@chakra-ui/react"
+import { Button, RestrictedGovtMasthead } from "@opengovsg/design-system-react"
+import { BiChevronLeft } from "react-icons/bi"
+
+import type { NextPageWithLayout } from "~/lib/types"
+import { PublicPageWrapper } from "~/components/AuthWrappers"
+import { FullscreenSpinner } from "~/components/FullscreenSpinner"
+import { IsomerLogo } from "~/components/Svg"
+import { SingpassLoginButton } from "~/features/sign-in/components"
+import { UnableToUseSignpassModal } from "~/features/sign-in/components/UnableToUseSignpassModal"
+import { SIGN_IN } from "~/lib/routes"
+import { trpc } from "~/utils/trpc"
+
+const SingpassSignInPage: NextPageWithLayout = () => {
+  const {
+    isOpen: isUnableToUseSingpassModalOpen,
+    onOpen: openUnableToUseSingpassModal,
+    onClose: closeUnableToUseSingpassModal,
+  } = useDisclosure()
+  const router = useRouter()
+  const { data, isLoading } = trpc.auth.singpass.getName.useQuery()
+
+  const handleBackToLogin = async () => {
+    await router.push(SIGN_IN)
+  }
```

Comment:
question: is this a pure frontend thing, or should we also clear the session so they can restart? im assuming when user click "Back to login/Not you", they would want to "logout" and restart the whole session

Reply:
Yup, this is purely frontend. If they accidentally click on this button, they can click on the browser back button to be brought back to the Singpass login page to continue where they have left off. If we clear the session here, the user would have to restart the email OTP process again.